addons/pharma/functions/fnc_treatmentAdvanced_Caffeine.sqf

```diff
@@ -0,0 +1,37 @@
+#include "script_component.hpp"
+/*
+ * Author: Miss Heda
+ * Begins Caffeine Treatment
+ *
+ * Arguments:
+ * 0: Medic <OBJECT>
+ * 1: Patient <OBJECT>
+ * 2: Body Part <STRING>
+ * 3: Treatment <STRING>
+ * 4: Item User (not used) <OBJECT>
+ * 5: Used Item <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, cursorObject, "RightArm", "Pervitin", objNull, "kat_pervitin"] call kat_pharma_fnc_treatmentAdvanced_Caffeine;
+ *
+ * Public: No
+ */
+
+params ["_medic", "_patient", "_bodyPart", "_classname", "", "_usedItem"];
+
+if (ACE_Player != _patient) then {
+
+	[LLSTRING(NoSelfTreatment), 2, _medic] call ACEFUNC(common,displayTextStructured);
```

Comment:
why this tho? i understand it

Reply:
Cause I want the player to know why it won't work if they give it to someone. Otherwise they'll report that as a bug.

Reply:
sounds kinda "stupid" that trooper cant take caffeine or pervitin by himself